Content.Server/Doors/Systems/AirlockSystem.cs

```diff
@@ -24,6 +27,14 @@ public override void Initialize()
         SubscribeLocalEvent<AirlockComponent, ActivateInWorldEvent>(OnActivate, before: new[] { typeof(DoorSystem) });
     }
 
+    protected override void OnBeforeDoorOpened(EntityUid uid, AirlockComponent airlock, BeforeDoorOpenedEvent args)
+    {
+        base.OnBeforeDoorOpened(uid, airlock, args);
+
+        if (args.User.HasValue)
+            _forensicsSystem.ApplyEvidence(args.User.Value, uid);
```

Comment:
this should just be in the code for doors, as opening any type of doors--not just airlocks--should leave evidence. This code should be moved into `SharedDoorSystem.StartOpening()`.

it should also be in the closing code too, if you manually close the door instead of letting it shut normally.